content/content-tool/tool/src/java/org/sakaiproject/content/tool/ResourcesAction.java

```diff
@@ -3125,10 +3125,13 @@ else if(contentService.isPubView(resource.getId()))
 		{
 			siteId = ref.getContext();
 		}
-		else
+		
+		// id may be null in format of /group/<site_id>, which leads to null value for the reference context field
+		if (siteId == null && ToolManager.getCurrentPlacement() != null)
```

Comment:
null value of siteId won't make SiteService.allowUpdateSite() call fail. Nevertheless, I added a routine there to get the site id from collection reference string.